ReactWindows/ReactNative.Shared/Modules/I18N/I18NUtil.cs

```diff
@@ -47,19 +62,31 @@ public static bool IsRightToLeftForced
         {
             get
             {
+#if WINDOWS_UWP
                 return (bool?)ApplicationData.Current.LocalSettings.Values[ForceRTL] ?? false;
+#else
+                return Boolean.Parse(ConfigurationManager.AppSettings[ForceRTL]);
+#endif
             }
             set
             {
+#if WINDOWS_UWP
                 ApplicationData.Current.LocalSettings.Values[ForceRTL] = value;
+#else
+                ConfigurationManager.AppSettings[AllowRTL] = value.ToString();
+#endif
             }
         }
 
         private static bool IsDeviceRightToLeft
         {
             get
             {
+#if WINDOWS_UWP
                 return ResourceContext.GetForCurrentView().QualifierValues["LayoutDirection"] == "RTL";
+#else
+                return CultureInfo.CurrentCulture.TextInfo.IsRightToLeft;
+#endif
```

Comment:
Is this a x-plat solution (i.e., do you know if this would work for UWP as well)?